Test.c

```diff
@@ -1,3 +1,38 @@
+#include <stdio.h>
+#include<stdlib.h>
+#include "Priority_Queue.h"
 #include "Test.h"
 
+void Test (void){
```

Comment:
@ManarAbdelraouf : You have to change this function name e.g `Test` to specific test name e.g `Test_PriorityQueue`